dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/sql/SqlTask.java

```
@@ -182,7 +183,7 @@ public void handle() throws Exception {
                 try {
                     con.close();
```

Did you eat the exception?